src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/DiaryOverviewTableViewController.swift

```diff
@@ -133,7 +123,16 @@ class DiaryOverviewTableViewController: UITableViewController {
 		guard let cell = tableView.dequeueReusableCell(withIdentifier: String(describing: DiaryOverviewDayTableViewCell.self), for: indexPath) as? DiaryOverviewDayTableViewCell else {
 			fatalError("Could not dequeue DiaryOverviewDayTableViewCell")
 		}
-		cell.configure(cellViewModel: viewModel.cellModel(for: indexPath))
+		cell.configure(
+			cellViewModel: viewModel.cellModel(for: indexPath),
+			didTapClickableView: { [weak self] in
+				guard let self = self else {
+					Log.error("Create strong self failed. Could not open day.")
+					return
+				}
+				self.onCellSelection(self.viewModel.day(by: indexPath))
```

Comment:
I think its unlikely in this case, but the index path of the cell could change after other cells getting removed or rearranged. A solution is to load the current indexPath for the cell like this `let currentIndexPath = self.tableView.indexPath(for: cell)`

Reply:
Good point. I added it.